portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path

```diff
@@ -63,6 +63,11 @@
 	<td>//li[contains(@class,'list-group-item')]//div[contains(@class,'justify-content-center')]//p[(@class="list-group-title") and contains (.,'${entryName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>VIEW_ENTRY_NAME</td>
+	<td>//li[contains(@class,"list-group-item list-group-item-flex") and contains(.,'${key_name}')]</td>
+	<td></td>
+</tr>
```

Comment:
I think `DATA_SET_VIEW_ENTRY_NAME` path does the same that `VIEW_ENTRY_NAME` if so, rename yours and delete the existing one.
```suggestion
<tr>
	<td>DATA_SET_VIEW_ENTRY_NAME</td>
	<td>//li[contains(@class,"list-group-item list-group-item-flex") and contains(.,'${key_name}')]</td>
	<td></td>
</tr>
```